velox/substrait/TypeUtils.h

```diff
@@ -14,11 +14,15 @@
  * limitations under the License.
  */
 
+#include <core/Expressions.h>
 #include "velox/type/Type.h"
 
 namespace facebook::velox::substrait {
 
 /// Return the Velox type according to the typename.
 TypePtr toVeloxType(const std::string& typeName);
 
+/// Return the Substrait type signature according to the type
+std::string substraitSignature(const TypePtr& type);
```

Comment:
Is this the opposite of the toVeloxType function above? If so, let's rename toSubstraitType.

To make this PR smaller and easier to remove, please, extract this new function and its test into a separate PR.

Reply:
No, it only convert to substrait type signature which can't be convert backed to velox type,  for more detail information please refer to substrait signature [here](https://substrait.io/extensions/#function-signature-compound-names)